gb-dbg/src/debugger/flow_control.rs

```diff
@@ -5,17 +5,20 @@ use std::ops::ControlFlow;
 pub struct FlowController;
 
 impl FlowController {
-    pub fn draw(&self, ui: &mut Ui) -> Option<ControlFlow<(), RunDuration>> {
-        let mut ret: Option<ControlFlow<(), RunDuration>> = None;
+    pub fn draw(&self, ui: &mut Ui) -> Option<ControlFlow<RunDuration>> {
+        let mut ret: Option<ControlFlow<RunDuration>> = None;
         ui.horizontal(|ui| {
             if ui.button("Step").clicked() {
-                ret = Some(ControlFlow::Continue(RunDuration::Step));
+                log::error!("clicked on step");
+                ret = Some(ControlFlow::Break(RunDuration::Step));
```

Comment:
Why break?
Defined behaviour was: On click, Continue execution

Reply:
For me, it was the other way around:
> click => Break after step / frame / 1 sec

This allow to return `Continue` when we want to resume/clear scheduled stop